traffic_portal/app/src/common/modules/form/deliveryServiceStaticDnsEntry/form.deliveryServiceStaticDnsEntry.tpl.html

```diff
@@ -57,9 +57,20 @@
                 </div>
             </div>
             <div class="form-group" ng-class="{'has-error': hasError(dsStaticDnsEntryForm.address), 'has-feedback': hasError(dsStaticDnsEntryForm.address)}">
-                <label for="address" class="control-label col-md-2 col-sm-2 col-xs-12">Address *</label>
+                <label for="address" class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12">Address *<div class="helptooltip">
+                    <div class="helptext">The Address Rules.
+                        <ul>
+                            <li>Type:A_RECORD, address should be an IPv4 address.</li>
+                            <li>Type:AAAA_RECORD, address should be an IPv6 address.</li>
+                            <li>Type:CNAME, address must end with a trailing period.</li>
```

Comment:
can you change this to:

`Type:CNAME, address must be a valid DNS name ending with a trailing period.

Reply:
sigh.... done